lua/telescope/actions/init.lua

```diff
@@ -144,10 +144,14 @@ function actions.center(_)
   vim.cmd(':normal! zz')
 end
 
-function actions.goto_file_selection_edit(prompt_bufnr)
+function actions.select(prompt_bufnr)
   actions._goto_file_selection(prompt_bufnr, "edit")
 end
 
+function actions.goto_file_selection_edit(prompt_bufnr)
+  actions.select(prompt_bufnr)
+end
```

Comment:
actions.goto_file_selection_edit = actions.select

Reply:
That makes sense. I'll add that change along with aliases for all the other functions.